models/topics.js

```diff
@@ -54,4 +55,43 @@ exports.selectCommentsByArticleId = (article_id) => {
     });
 };
 
+exports.addComment = (article_id, username, body) => {
+    if (!username || !body) {
+        return Promise.reject({ status: 400, msg: 'missing object properties' })
+    } else {
+        return db
+        .query(`INSERT INTO users (username, name) VALUES ($1, $2) RETURNING*;`, [username, 'name'])
+        .then(() => {
+        return checkArticleExists(article_id)
```

Comment:
No need to check if the article exists here really - just try and insert the comment and the database will tell you!